# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../e2fsprogs/subst-stat.patch.cross
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- e2fsprogs-1.43.4/util/subst.c.vanilla	2017-04-23 12:48:02.564763193 +0000
+++ e2fsprogs-1.43.4/util/subst.c	2017-04-23 12:48:11.348763344 +0000
@@ -22,9 +22,7 @@
 #ifdef HAVE_SYS_TYPES_H
 #include <sys/types.h>
 #endif
-#ifdef HAVE_SYS_STAT_H
 #include <sys/stat.h>
-#endif
 #include <fcntl.h>
 #include <time.h>
 #include <utime.h>
